doc/DHCPv6_relay/DHCPv6-relay-agent-High-Level-Design.md

```diff
@@ -172,27 +172,90 @@ DHCP|intf-i|dhcpv6_option|rfc6939_support: &quot;true&quot;
 
 sonic-dhcpv6-relay.yang
 <pre>
-module DHCP  
-    container DHCP {  	
-        list VLAN_LIST {
-    		key name;
-   		    leaf name {
-    			type string;
-  		    }
-   		    leaf dhcpv6_servers {
-     		    	type inet6:ip-address;
-  		    }
-		    leaf dhcpv6_option|rfc6939_support {
-			    type bool;
-		    }
+module sonic-dhcpv6-relay {
+
+	namespace "http://github.com/Azure/sonic-dhcpv6-relay";
+
+	prefix sdhcpv6relay;
+
+	yang-version 1.1;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	organization "SONiC";
+
+	contact "SONiC";
+
+	description "DHCPv6 Relay yang Module for SONiC OS";
+
+	revision 2021-10-30 {
+		description "First Revision";
+	}
+
+	container sonic-dhcpv6-relay {
+
+		container DHCP_RELAY {
+
+			description "DHCP_RELAY part of config_db.json";
+
+			list DHCP_RELAY_LIST {
+
+				key "name";
+        
+				leaf name {
+					type string;
+				}
+
+        leaf-list dhcpv6_servers {
+                description "Configure the dhcpv6 servers";
+                type inet:ipv6-address;
         }
+
+        leaf dhcpv6_option|rfc6939_support {
+                description "Set rfc6939 for the relay";
+        		    type bool;
+		    }
+
+        leaf dhcpv6_option|interface_id {
+                description "Set interface-id for the relay";
+        		    type bool;
+		    }
     }
+  }
 }
 </pre>
 
 # Option 79 for client link-layer address
 
-Option 79 should be enabled by default and can be disabled through command line.
+Option 79 should be enabled by default and can be disabled through editing config_db.json and reapplying configuration or config database directly by the following commands:
+
+Usage:
+```
+config vlan dhcp_relay add <vlan_id> <dhcp_relay_destination_ip>
+redis-cli -n 4 hset DHCP_RELAY|<vlan> dhcpv6_option|rfc6939_support <true/false>
```

Comment:
i think it's better to mention here that restarting the dhcp_relay service is required for the changes to take effect.

Reply:
Also, why do we need to edit the VLAN table for dhcpv6relay? I thought we migrated to DHCP_RELAY table entirely on all labels. Please also mention this somewhere that all we required for DHCPv6 Relay sits under DHCP_RELAY table.

Reply:
Updated. Some older versions use VLAN table so it is good to update both for the time being

Reply:
Got it